pkg/jx/cmd/get_applications.go

```diff
@@ -151,6 +154,19 @@ func (o *GetApplicationsOptions) generateTable(apps []string, envApps []EnvApps,
 	appEnvMap := map[string]map[string]*ApplicationEnvironmentInfo{}
 	for _, appName := range apps {
 		row := []string{appName}
+		jxClient, ns, err := o.JXClientAndDevNamespace()
```

Comment:
I would create the client outside of the loop since a new client is not required for each iteration. Also I would return an error instead of just logging a warning. This looks like a fatal error.

Reply:
Agree on moving it out of the loop, I'll fix that. As far as the warning goes, I initially thought there might be a situation where the app name doesn't match the repo name, in which case I didn't want the command to fail. But actually there's no way for this to happen that I know of, so maybe you're right.

Reply:
Actually never mind, I was thinking about the git URL error message. This is definitely a fatal error.